src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -474,4 +472,11 @@ boolean passDataAsHex(int colNum) {
         return (JDBCType.BINARY == getColumn(colNum).getJdbctype() || JDBCType.VARBINARY == getColumn(colNum).getJdbctype()
                 || JDBCType.LONGVARBINARY == getColumn(colNum).getJdbctype());
```

Comment:
I know this didn't get changed, but would be better if we just do getColumn(colNum).getJdbctype() once

Reply:
cool, done